src/ip-fragment.c

```diff
@@ -150,18 +205,37 @@ static int ip_fragment_init(struct pg_brick *brick,
 	ip_fragment_config =
 		(struct pg_ip_fragment_config *) config->brick_config;
 
+	if (ip_fragment_config->mtu_size % 8) {
+		*errp = pg_error_new("mtu must be a  multiplier");
+		return -1;
+	}
 	brick->burst = ip_fragment_burst;
 	state->output = ip_fragment_config->output;
 	state->mtu_size = ip_fragment_config->mtu_size;
+	state->tbl = rte_ip_frag_table_create(16, 1024, 16 * 1024,
+					      (rte_get_tsc_hz() + MS_PER_S - 1)
+					      / MS_PER_S * MAX_FLOW_TTL,
+					      SOCKET_ID_ANY);
```

Comment:
You need to add some explanations here.
Could you add a macro to convert from seconds to cycles ?